src/components/GeneralLookupField.tsx

```diff
@@ -177,21 +178,24 @@ const GeneralLookupField = ({
           name={name}
           value={value}
           options={options}
+          inputValue={inputValue.current}
+          onInputChange={handleInputChange}
           onSelect={(option: IReceiverAddress) => {
-            setFieldValue(name, option, true); //if this gets deleted, it no longer shows as selected on interface, would like to set only object keys that are needed instead of full object
-            setFieldTouched(name, true, false);
+            // if this gets deleted, it no longer shows as selected on interface,
+            // would like to set only object keys that are needed
+            // instead of full object
+            // setFieldValue(name, option, true);
```

Comment:
I think without this form validation isn't run?

Reply:
I have the same behaviour on beta and this PR.
I uncommented the lines and will deal with this in the refactor.